STYLEGUIDE.md

```diff
@@ -0,0 +1,738 @@
+# Style Guide
+
+* Use soft-tabs with a two space indent.
```

Comment:
I know this is from copy-paste, but does the bullets for each paragraph bother anyone else?